packages/flutter/lib/src/material/stepper.dart

```diff
@@ -151,6 +153,15 @@ class Stepper extends StatefulWidget {
   ///
   /// The length of [steps] must not change.
   final List<Step> steps;
+  
+  /// How the Stepper's scroll view should respond to user input.
+  ///
+  /// For example, determines how the scroll view continues to animate after the
+  /// user stops dragging the scroll view. 
+  ///
+  /// Defaults to [AlwaysScrollableScrollPhysics] but consider changing to 
```

Comment:
Why are we defaulting this to anything?  Why not just keep it null to retain existing behavior?

Reply:
At the time I thought it would have been the better option to do but yeah with further testing it's better to keep to null. My bad!